### PR TITLE
Disable HTTP/2 by default

### PR DESCRIPTION
**Pull request by brandur**
See #866, but once in a while when using HTTP/2, stripe-go sends an
empty request body which then results in a 400 back from the server.

We tracked the issue down to this one open on the Go language's HTTP/2
implementation:

https://github.com/golang/go/issues/32441

For now, the best thing we can do is disable HTTP/2 by default for new
integrations. When the bug is fixed, we can turn it back on again.

r? @dcarney-stripe
cc @stripe/api-libraries

### CODE REVIEW

File: stripe.go [hunk 1]
```diff
@@ -944,7 +945,43 @@ var appInfo *AppInfo
...
+// `HTTPClient` value when using `BackendConfig`.
+var httpClient = &http.Client{
+	Timeout: defaultHTTPTimeout,
+
+	// There is a bug in Go's HTTP/2 implementation that occasionally causes it
+	// to send an empty body when it receives a `GOAWAY` message from a server:
+	//
+	//     https://github.com/golang/go/issues/32441
+	//
+	// This is particularly problematic for this library because the empty body
```
Comment (dcarney-stripe) [inline stripe.go]: Great comment; thanks for capturing all the context in here.  It's my sincere hope that we can re-enable H/2 support soon, when that Go issue gets addressed.

**Comment by brandur-stripe** [comment]
Thanks Dylan!

File: stripe.go (hunk 1 above)
Comment (brandur-stripe) [reply stripe.go]: Thanks! And yep, totally +1.

Comment (brandur-stripe) [reply stripe.go]: Thanks! And yep, totally +1.

**Comment by brandur-stripe** [comment]
Released as 61.20.0.

**Pull request merged**